app/controllers/ems_storage_controller.rb

```diff
@@ -64,6 +66,9 @@ def record_class
   end
 
   def textual_group_list
+    group_list = @record.try(:textual_group_list)
+    return group_list if group_list
+
```

Comment:
This seems like a shortcut to me :confused:

Reply:
probably should be something like`
def textual_group_list
  return @record.send(: textual_group_list) if @record.respond_to?(: textual_group_list)
...
`
The idea was to add an ability to override property names in each manager.

Reply:
We decided to have a single set of groups for all storage managers for now. The meaning is that some managers will display empty values or 0 for some properties. I will remove this code and just add a storage system and storage resources to the textual_group_relationsgios in the app/helpers/ems_storage_helper/textual_summary.rb.

Let's discuss later if we want to allow for any managers to display their specific properties.

Reply:
No, the whole return-with-if seems to me as a shortcut, introducing a bad coding pattern and increasing complexity. These things can lead to a lot of headache in the future :confused:

Reply:
We moved this change to the separate PR (7325)
https://github.com/ManageIQ/manageiq-ui-classic/pull/7325.

If you agree that this functionality is required, suggestions regarding a more correct implementation will be gladly accepted.